samples/client/petstore/swift4/default/docs/StoreAPI.md

```diff
@@ -0,0 +1,206 @@
+
+# **getInventory**
+```swift
+    open class func getInventory(completion: @escaping ((_ data: [String:Int]?,_ error: Error?) -> Void))
+```
+
+Returns pet inventories by status
+
+Returns a map of status codes to quantities
+
+### Example 
+```swift
+// the following code samples are still beta. If you notice any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
+import PetstoreClient
+
+
+// Returns pet inventories by status
+StoreAPI.getInventory() { (response, error) in
+    guard error == nil else {
+        print(error)
+        return
+    }
+
+    if (response) {
+        dump(response)
+    }
+}
+```
+
+### Parameters
+This endpoint does not need any parameter.
+
+### Return type
+
+**[String:Int]**
+
+### Authorization
+
+[api_key](../README.md#api_key)
+
+### HTTP request headers
+
+ - **Content-Type**: Not defined
+ - **Accept**: application/json
+
+[[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
+
+# **getOrderById**
+```swift
+    open class func getOrderById(orderId: Int64, completion: @escaping ((_ data: Order?,_ error: Error?) -> Void))
+```
+
+Find purchase order by ID
+
+For valid response try integer IDs with value <= 5 or > 10. Other values will generated exceptions
+
+### Example 
+```swift
+// the following code samples are still beta. If you notice any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
+import PetstoreClient
+
+let orderId = 987 // Int64 | ID of pet that needs to be fetched
+
+// Find purchase order by ID
+StoreAPI.getOrderById(orderId: orderId) { (response, error) in
+    guard error == nil else {
+        print(error)
+        return
+    }
+
+    if (response) {
+        dump(response)
+    }
+}
+```
+
+### Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **orderId** | **Int64**| ID of pet that needs to be fetched | 
+
+### Return type
+
+[**Order**](Order.md)
+
+### Authorization
+
+No authorization required
+
+### HTTP request headers
+
+ - **Content-Type**: Not defined
+ - **Accept**: application/xml, application/json
+
+[[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
+
+# **placeOrder**
+```swift
+    open class func placeOrder(body: Order, completion: @escaping ((_ data: Order?,_ error: Error?) -> Void))
+```
+
+Place an order for a pet
+
+### Example 
+```swift
+// the following code samples are still beta. If you notice any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
+import PetstoreClient
+
+let body = Order(123, 123, 123, "TODO", "status_example", 123) // Order | order placed for purchasing the pet
```

Comment:
- The initializer should have labels for parameters. 
- It looks `Bool` turns `Int`, should be `true` or `false`.

As is:
```swift
let body = Order(123, 123, 123, "TODO", "status_example", 123)
```
Ideal:
```swift
let body = Order(id: 123, petId: 123, quantity: 123, shipDate: "TODO", status: "status_example", complete: true)
```